workspaces/liferay-blank-workspace/gradle.properties

```diff
@@ -0,0 +1,12 @@
+##
+## Check GETTING_STARTED.markdown for additional properties and their default
+## values.
+##
+
+liferay.workspace.bundle.dist.include.metadata=true
+liferay.workspace.modules.dir=modules
+liferay.workspace.node.package.manager=yarn
```

Comment:
Are we wanting to make yarn the new default for new `blade init` workspaces?

Reply:
I think we can parameterize this one, or strip it out by default.